### PR TITLE
fixed typos

### PR DESCRIPTION
**Pull request by Mackenzie-OO7**
Fixed typos and punctuation errors in the **Best Practices** and **Operations** page.

### CODE REVIEW

File: src/topics/best-practices.md [hunk 1]
```diff
@@ -56,27 +56,27 @@ all are required.
   known to work with this description under `SoftwareRequirement`.
 
 - `format` should be specified for all input and output `File`s.
-  Bioinformatics tools should use format identifiers from [EDAM][edam-example].
+  Bio-informatics tools should use format identifiers from [EDAM][edam-example].
```
Comment (kinow) [inline src/topics/best-practices.md:59]: I think Bioinformatics is OK too. I see it more frequently than bio-informatics

Comment (kinow) [reply src/topics/best-practices.md:59]: And just a few words down there's non-bioinformatics, not non-bio-informatics. For consistency I think bioinformatics would be better here.

File: src/topics/best-practices.md [hunk 2]
```diff
@@ -94,7 +94,7 @@ all are required.
   should utilise the [`SubworkflowFeatureRequirement`][subworkflow] to make their
   workflow modular and allow sections of them to be easily reused.
 
-- Software containers should be made to be conformant to the ["Recommendations for the packaging and containerizing of bioinformatics software"][containers] (also useful to other disciplines).
+- Software containers should be made to be conformant to the ["Recommendations for the packaging and containerizing of bio-informatics software"][containers] (also useful to other disciplines).
```
Comment (kinow) [inline src/topics/best-practices.md:97]: Ditto above, bioinformatics rather than bio-informatics.

**Comment by Mackenzie-OO7** [comment]
@kinow Thanks for the review!

**Comment by kinow** [comment]
> @kinow Thanks for the review!

Thanks for fixing it so quickly @Mackenzie-OO7 !

**Comment by swzCuroverse** [comment]
@tetron ready to merge.  Thank you.

**Review by tetron — APPROVED** [review]

**Pull request merged**